starter-code/app.js

```diff
@@ -2,4 +2,42 @@ const express = require('express');
 const app = express();
 const Chuck  = require('chucknorris-io');
 const client = new Chuck();
+const expressLayouts = require('express-ejs-layouts');
 
+
+app.set('views', __dirname + '/views');
+app.set('view engine', 'ejs');
+app.set('layout', 'views/index')
+
+app.use(express.static('static'));
+
+app.get('/', (request, response, next) => {
+  response.send('chuck norris');
```

Comment:
Here, in fact, you want to display the same thing that in /random (not only the index but with a joke), for doing that:
```
res.redirect('/random');
```